flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/internal/AbstractJdbcOutputFormat.java

```diff
@@ -72,15 +70,7 @@ public void close() {
 	}
 
 	private void closeDbConnection() {
```

Comment:
Maybe we don't need this method anymore, we can directly invoke `connectionProvider.closeConnection` in `close()`. 

The same to `establishConnection`.

Reply:
`closeDbConnection` and `establishConnection` is removed in fixup commit.